packages/tools/src/apm.coffee.md

```diff
@@ -12,7 +12,7 @@ Install Atom packages with APM.
 
 ## Source code
 
-    module.exports = ({options}) ->
+    module.exports.install = ({options}) ->
```

Comment:
Where did you get your inspiration from, not a single action is exposing a property instead of `module.exports`